src/badguy/haywire.hpp

```diff
@@ -47,6 +47,8 @@ class Haywire final : public WalkingBadguy
   virtual std::string get_display_name() const override { return display_name(); }
   virtual bool is_snipable() const override { return true; }
 
+  inline bool is_exploding() const { return m_is_exploding; }
```

Comment:
Unneeded `inline`.

Reply:
If so, it could be a separate PR which does that for every getter in the code.

Reply:
that'd make sense

Reply:
inline has no benefit. Why did you think it would?

Reply:
Depends on optimization mode (and on compiler) but that what @MatusGuy said is mostly true.

Reply:
mrkubax knows anything therefore this should be a real pull request 😎

Reply:
So every getter should be inline

Reply:
Well it probably doesn't have impact worth the effort.